webapp/packages/plugin-data-viewer/src/TableViewer/TableError.module.css

```diff
@@ -28,7 +28,6 @@
 .errorBody {
   display: flex;
   gap: 24px;
-  align-items: center;
```

Comment:
it looks weird if everything is centered. In case the error is huge and scrollable, the error icon is in the middle, which is not visible for the user to indicate it is an error. image with the fixed case attached

<img width="933" alt="Снимок экрана 2024-07-22 в 21 51 59" src="https://github.com/user-attachments/assets/b43f4b17-7e9c-42a0-96ae-d8c7f9c08ef9">